frontend/src/js/preview-v2/HeadlineStats.tsx

```diff
@@ -52,26 +40,25 @@ export default function HeadlineStats({ statistics }: HeadlineStatsProps) {
     }
     return t("preview.dateError");
   };
-
-  const { total, dateRange } = statistics;
-
+  const dateRange = statistics?.dateRange;
+  
   return (
     <Root>
       <MetaValue>
         <Key>Zeilen:</Key>
-        {total}
+        {statistics?.total || "Noch nicht geladen"}
```

Comment:
maybe find a i18next string for that.
Now that I think about it, we should consider extracting all the strings (Zeilen, Min Datum, etc.) to i18next language files.
What do you think?

Reply:
We will recode the Headline Stats in the next Step - so I don't think that is necessary rn